firestore-bigquery-export/CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## Version 0.1.45
+
+feature - bring back the backfill parameter
+
+feature - add a new parameter EXCLUDE_OLD_DATA to reduce payload size
+
+feature - add a new parameter DATABASE_ID to support Firestore database selection
```

Comment:
I didn't merge this one, prefer to put it on a seprate release so will not merge